### PR TITLE
Adding code to the use model dialogue

### PR DESCRIPTION
**Pull request by stevekogo**

### CODE REVIEW

**Comment by stevekogo** [comment]
This Fixes #1766 However, the formula preview is not working as intended, we should have a discussion with @dannyparsons to fix this. @rdstern, could you please give suggestions on what should be there on the predict subdialogue?

File: instat/dlgTwoVariableUseModel.vb [hunk 1]
```diff
@@ -14,8 +14,95 @@
...
+            Model()
+        Else
+            ucrBaseUseModel.OKEnabled(False)
+        End If
+    End Sub
+
+    Private Sub Model()
+        If Not ucrReceiverUseModel.IsEmpty And Not ucrSelectorUseModel.ucrAvailableDataFrames.cboAvailableDataFrames.SelectedItem = "" Then
+            clsRModel.SetRCommand("formula")
+            clsRModel.AddParameter("", clsRFunctionParameter:=clsRCommand)
```
Comment (dannyparsons) [inline instat/dlgTwoVariableUseModel.vb]: The parameter name should be `x` I think from looking at the documentation for `formula`.

File: instat/dlgTwoVariableUseModel.vb [hunk 2]
```diff
@@ -75,8 +75,8 @@ Public Class dlgTwoVariableUseModel
     Private Sub Model()
         If Not ucrReceiverUseModel.IsEmpty And Not ucrSelectorUseModel.ucrAvailableDataFrames.cboAvailableDataFrames.SelectedItem = "" Then
             clsRModel.SetRCommand("formula")
-            clsRModel.AddParameter("", clsRFunctionParameter:=clsRCommand)
-            'strDataTypes = frmMain.clsRLink.RunInternalScriptGetValue(clsRModel.ToScript()).AsCharacter.AsList(0).ToString
+            clsRModel.AddParameter("x", clsRFunctionParameter:=clsRCommand)
+            'strModel = frmMain.clsRLink.RunInternalScriptGetValue(clsRModel.ToScript()).AsCharacter().ToList().ToString()
```
Comment (stevekogo) [inline instat/dlgTwoVariableUseModel.vb]: @dannyparsons The commented line is intended to run the script internally and display the model in the formula preview control. However, this is not working. Can you help me fix it? We can have a skype call to handle this.

Comment (dannyparsons) [inline instat/dlgTwoVariableUseModel.vb]: Try `.AsCharacter(0)`. It depends what clsModel returns in R? If it's a string then this should work.

Comment (stevekogo) [inline instat/dlgTwoVariableUseModel.vb]: This does not work. We should discuss this soon.

Comment (dannyparsons) [inline instat/dlgTwoVariableUseModel.vb]: OK I see why this doesn't work now. The formula function returns a formula object, so it won't convert to string. 
One option is to instead run as.character(as.formula(x)) in R, but you will see this doesn't give the right string. Another is to use RunInternalScriptGetOutput

**Comment by stevekogo** [comment]
@dannyparsons, @rdstern this is now working apart from the button to the predict subdialogue. You can now merge it as we wait for @rdstern to suggest how the subdialogue should look like.

**Comment by dannyparsons** [comment]
Thanks, we'll review and get back to you

**Pull request merged**